### ISSUE TITLE
Routers are not being generated

### ISSUE THREAD

**Issue by jeffdoubleyou** [beego/bee#800]
I am not sure if routers are supposed to be automatically generated in v2 any longer, but they don't seem to be generated for me when running my application.

I see some reference to a new bee generate command "bee generate routers", but this doesn't seem to exist in bee v2.0.2.

How do you generate the routers/commentsRouter.go file using v2?  I cannot seem to find any reference to this in the documentation or any pathway into this from the code.

What am I missing?

Thank you.

**Comment by flycash** [comment]
Yes, it's in develop branch. Please update your Bee version to newest develop commit id
